### ISSUE TITLE
Surge XT installation does not install factory content or presets on an alternative Drive

### ISSUE THREAD

**Issue by JeffGreenlee42** [surge-synthesizer/surge#6290]
- I have a Mac Mini M1.. installed with Montery version 12.4
- I installed Surge XT (tried both main one and Beta 1.1 - Same issue for each). 
- I choose All the installation options  and when I get to selecting a drive I choose a second drive (not the main)  
- Both drives are SSD though and have plenty of space.  
- After installation completes,  Installation box says installation was successful!  No errors reported. 
- I open Surge XT either as Standalone Or as VST (Ableton Live 11 or Bitwig 4.3.3).  
- I see Surge looks fine ...  However If I try to access Factory content (menu that opens when clicking on browser).."Open Factory Patches Folder"  there is Nothing. no finder window opens ... no browser. 
- If I click on left and right arrows next to "patch".  or "Category" in browser, nothing happens. The only patch in existence is "Init" apparently.
- I look at "About Surge" window and I see path listed for "Factory Data":  /Library/Application Support/Surge XT. 
- That PATH does not exist!  I checked both drives  on my Mac AND even my Mac/Users/Library. ...  "Applicaiton Support" folder lives in all three locations But there is no Folder "Surge XT" that  exists within Applciation Support!

One small detail: After installation,  In order to see Surge within Ableton and/or Bitwig, I had to create a symbolic links to point to VST3 files , since they exist in different location from where Main System wide VST3's live. This is common practice to make all VST's and VST3 files accessible from one folder.   Surge opens fine as VST's in my DAWS,  It just has zero factory or 3rd party patches or wavetables. 

Version: Surge XT 1.1.nightly.946a890
Build: 2022-06-12 @ 22:51:40 on 'Mac-1655062102988.local/pipeline' with 'AppleClang-12.0.0.12000032' using JUCE 6.1.6
System: macOS 64-bit Standalone on Apple M1
Sample Rate: 48.0 kHz

![530CECC7-D835-4FD4-9E5A-A42A4E9EE203_4_5005_c](https://user-images.githubusercontent.com/22585181/174429569-9e0a806f-98c6-4eae-983a-59498d9ce330.jpeg)

**Comment by JeffGreenlee42** [comment]
Ok.. so  I definitely confirmed this is a bug.   I removed all Surge components from 2nd drive and then installed same version on Main Macintosh drive ..and Everything is installed as expected.   
So I guess I would suggest one of 2 things happen?  
1).  Fix the issue?  So that when Mac installation occurs .. All content is installed.. regardless of which (local) Drive the Plugin is installed on.    
OR
2). Remove the "alternative installation drive" option from Mac Installation steps. so that users save some hair (being pulled out!) - and just state that only main drive installation is allowed.

**Comment by mkruselj** [comment]
@baconpaul Worth pushing this to 1.1.n? Or maybe even 1.1?

**Comment by baconpaul** [comment]
Definitely not 1.1

I’ll take a look for 11n

**Comment by baconpaul** [comment]
So for our 1.1.1 release I'll go with the 'dont provide the destination' option; I can't quite see how to get application support on another drive to work easily. i'll keep this issue open but bounce it back into 1.1.n now i have it so it doesn't ask me any more.

**Comment by mkruselj** [comment]
@baconpaul Maybe our modern tools could help with this one, what do you say? 🙂

**Comment by baconpaul** [comment]
maybe. the problem is when you pick another drive i need to know that somehow. i don't know what that mechanism would be. probably some form of application setting but where to put that.

**Comment by mkruselj** [comment]
Or, we could simply just not allow it at all. We don't allow it on Windows either, stuff just goes to ProgramData.

Then we could close this issue. 🙂

**Comment by baconpaul** [comment]
right. and the current installers - if you have remapped the location of /Library - will follow that.
